src/Nataliia/HelloWorld.java

```diff
@@ -0,0 +1,7 @@
+package Nataliia;
+
+public class HelloWorld {
```

Comment:
looks good Nataliia, but please go back and add an '!' at the end to show pure excitement to the world